app/services/flags/create_flag.rb

```diff
@@ -3,6 +3,7 @@
 class Flags::CreateFlag
   include Service::Base
 
+  policy :invalid_access
```

Comment:
Moving this policy as the first step since it only relies on `guardian` and will stop the service if the user can’t create a flag.